lib/sucker_punch/job.rb

```diff
@@ -32,17 +32,17 @@ def logger
     end
 
     module ClassMethods
-      def perform_async(*args)
+      def perform_async(*args, **kwargs)
         return unless SuckerPunch::RUNNING.true?
         queue = SuckerPunch::Queue.find_or_create(self.to_s, num_workers, num_jobs_max)
-        queue.post(args) { |job_args| __run_perform(*job_args) }
+        queue.post([args, kwargs]) { |job_args, job_kwargs| __run_perform(*job_args, **job_kwargs) }
```

Comment:
```ruby
queue.post { __run_perform(*args, **kwargs) }
```

I think, that this version will work in the same way. I'll need to double-check implementation in concurrent-ruby.

Reply:
I've tracked down argument forwarding to this place: https://github.com/ruby-concurrency/concurrent-ruby/blob/master/lib/concurrent-ruby/concurrent/executor/ruby_thread_pool_executor.rb#L351

Both `args` and `job_args` are references to the same object.